cmd/clusterctl/client/cluster/mover.go

```diff
@@ -43,6 +43,7 @@ type ObjectMover interface {
 type objectMover struct {
 	fromProxy             Proxy
 	fromProviderInventory InventoryClient
+	isDryrun              bool
```

Comment:
Same as above only with s/clusterClient/objectMover (should we consider to scope down the change to the single command?)